### ISSUE TITLE
Invalid bundle with Xcode 15.3

### ISSUE THREAD

**Issue by jatindervrify** [aws/amazon-chime-sdk-ios#655]
**Describe the bug**
When archiving or deploying a build with `AmazonChimeSDKMachineLearning` as part of an SPM package an error occurs. This is only on Xcode 15.3 and not previous versions of Xcode. Unfortunately I'm not sure if this is an Xcode issue or a Chime SDK issue. Everything works fine on Xcode 15.2. 

**To Reproduce**

0. Include Chime in any SPM package as a dependency. Ex: 
```
    .... 
    dependencies: [
        .package(url: "https://github.com/aws/amazon-chime-sdk-ios-spm", exact: "0.23.3"),
         ....
    ],
    targets: [
        // Targets are the basic building blocks of a package, defining a module or a test suite.
        // Targets can depend on other targets in this package and products from dependencies.
        .target(
            name: "Chime",
            dependencies: [
            ...
                .product(name: "AmazonChimeSDK", package: "amazon-chime-sdk-ios-spm"),
                .product(name: "AmazonChimeSDKMedia", package: "amazon-chime-sdk-ios-spm"),
                .product(name: "AmazonChimeSDKMachineLearning", package: "amazon-chime-sdk-ios-spm"),
           ...
            ]),
```
1. Using Xcode 15.3 simply deploy your build to a device or attempt to upload to TestFlight. The following error will occur:

```
Invalid bundle structure. The “___.app/Frameworks/AmazonChimeSDKMachineLearning.framework/AmazonChimeSDKMachineLearning” binary file is not permitted. Your app cannot contain standalone executables or libraries, other than a valid CFBundleExecutable of supported bundles. For details, visit: https://developer.apple.com/documentation/bundleresources/placing_content_in_a_bundle (ID: e1ef147f-d52e-46c3-be97-e2bf32fb63bd)
```

**Expected behavior**
The `AmazonChimeSDKMachineLearning` binary should be valid for use with Xcode. Removing the `AmazonChimeSDKMachineLearning` results in everything working but then we can no longer use the AWS blur. 

**Logs**
N/A

**Screenshots**
<img width="655" alt="image" src="https://github.com/aws/amazon-chime-sdk-ios/assets/41025959/1ee14548-f057-48aa-9e9d-675e61f4d189">

**Test environment Info (please complete the following information):**
 - Device: Any Simulator and Physical devices
 - OS: iOS 17.4/17.2
 - Version AmazonChimeSDK: 0.23.3
 - Version AmazonChimeSDKMedia: ^
 - Can you reproduce this in the demo app? Can't build it
**If you are reporting a crash:**
 - How often is this crash occurring?
 - Are there metrics on CPU or memory use at the time of the crash?
 - Are you using bitcode supported binaries?
 - Please provide full crash logs in the form of a ".crash" or ".ips" file with a binary images section.

**Additional context**
N/A

**Comment by bukira** [comment]
Same, still not fixed in 0.25.0 

Failed to load Info.plist from bundle at path /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.KDeZ6n/extracted/name.app/Frameworks/AmazonChimeSDKMachineLearning.framework; Extra info about "/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.KDeZ6n/extracted/nname.app/Frameworks/AmazonChimeSDKMachineLearning.framework/Info.plist": Couldn't stat /var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.KDeZ6n/extracted/name.app/Frameworks/AmazonChimeSDKMachineLearning.framework/Info.plist: No such file or directory

**Comment by georgezy-amzn** [comment]
Hi all, sorry for the inconvenience, we are working on a fix, will release it asap.

**Comment by jatindervrify** [comment]
@georgezy-amzn any updates on this one? This continues to be an issue for us even with the new Xcode 15.4 beta that has just been released.

**Comment by bukira** [comment]
any more on this?

**Comment by bukira** [comment]
any update?

**Comment by georgezy-amzn** [comment]
Hi all, the fix is coming very soon.

**Comment by jatindervrify** [comment]
@georgezy-amzn Thanks for getting the fix released! Any idea when `0.25.1` will be added to https://github.com/aws/amazon-chime-sdk-ios-spm?

**Comment by georgezy-amzn** [comment]
Hi all, the fix has been released with 0.25.1 on SPM now.
